src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -44,28 +48,41 @@ public Command parseCommand(String userInput) throws ParseException {
         final String arguments = matcher.group("arguments");
         switch (commandWord) {
 
+        case Command.PATRON_COMMAND_GROUP:
+            // TODO : ADITI please do something like "return new PatronCommandParser().parse(arguments);"
+            return null;
+
+        case Command.BOOK_COMMAND_GROUP:
+            return new BookCommandParser().parse(arguments);
+
+        // TODO : ADITI please move add, edit, del, find, list into your new PatronCommandParser class in new package
+        // TODO : CHANGE TO "case ADD_COMMAND_WORD", import static "ADD_COMMAND_WORD" from Command class
```

Comment:
Good job on separating the two different commands and introducing a new abstraction layer.